salt/profiles/config/srv-profiles-app.cfg

```diff
@@ -1,5 +1,6 @@
 [profiles]
 environment = {{ pillar.elife.env }}
+db = postgresql://{{ pillar.profiles.db.username }}:{{ pillar.profiles.db.password }}@{{ salt['elife.cfg']('cfn.outputs.RDSHost') or pillar.profiles.db.host }}:{{ salt['elife.cfg']('cfn.outputs.RDSPort') or pillar.profiles.db.port }}/{{ salt['elife.cfg']('project.rds_dbname') or pillar.profiles.db.name }}
```

Comment:
This is messy. Is there a nicer way to construct a URI here? Or maybe make the app do it?

Reply:
I guess that:
- this is a standard format for the connection string to the database
- we don't want to replicate this in every project that uses (or don't) a Postgresql RDS database
`salt['elife.cfg']` comes from https://github.com/elifesciences/builder-base-formula/blob/master/_modules/elife.py#L98 so we may extract a similar method there? It could be single values like `db_host()` or the whole `db_postgresql_connection_string()`. This depends on the following suggestion of turning to a generic `pillar.elife.db`